torch/_dynamo/variables/constant.py

```diff
@@ -110,13 +110,6 @@ def unpack_var_sequence(self, tx):
             raise NotImplementedError from e
 
     def const_getattr(self, tx: "InstructionTranslator", name):
-        if isinstance(self.value, type):
-            raise UserError(
-                UserErrorType.ANTI_PATTERN,
-                "Can't access members of type(obj) for a generated custom object. "
-                "Please use __class__ instead",
-                case_name="type_reflection_method",
-            )
```

Comment:
This is also no longer possible after [#141504](https://github.com/pytorch/pytorch/pull/141504/files#diff-037b8614f7f7e02131cebf2d5a9704907e6d65b6fec5973177dc4f18aa834b3cR55-R65)